celery/app/trace.py

```diff
@@ -623,3 +622,7 @@ def __protected_call__(self, *args, **kwargs):
             return orig(self, *args, **kwargs)
         BaseTask.__call__ = __protected_call__
         BaseTask._stackprotected = True
+
+
+def _info(fmt, context):
```

Comment:
I would suggest to move this function at the top of the file and retain the name `info`, thus keeping the changes to a minimum.
Also, I think you need to add documentation in the function docstring in order to resolve the Travis CI errors.